### PR TITLE
Add a Klallam (clm) g2p mapping

### PR DESCRIPTION
**Pull request by joanise**
Pull request template for adding a new language
-----------------------------------------------

*Do not use this for other types of pull requests*

* **Please check if the PR fulfills these requirements**
- [x ] Mapping files are added in `g2p/mappings/langs`
- [x ] Mapping is either added to an existing folder or a new folder has been added
- [x ] Language folder and files use appropriate [ISO 639-3 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-3_codes)
- [x ] Config.yaml file includes all author names, and settings necessary
- [x ] Please add some test data in `g2p/tests/public/data`. The added file should be a csv/tsv/psv file and each row should have the format `[input_mapping_code,output_mapping_code,input_string,output_string]`
- [x ] As the last step, G2P has been updated by running `g2p update` locally and committing the change
- [x ] You agree to license your contribution under the same license as this project (see [LICENSE](https://github.com/roedoejet/g2p/blob/main/LICENSE) file).

* **Other information**:

### CODE REVIEW

File: g2p/mappings/langs/clm/clm_equiv.csv [hunk 1]
```diff
@@ -0,0 +1,15 @@
+’,̕
```
Comment (roedoejet) [inline g2p/mappings/langs/clm/clm_equiv.csv:1]: not really reviewing these all but seems plausible

File: g2p/mappings/langs/generated/clm-ipa_to_eng-ipa.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    {"in": "a", "out": "æ"},
+    {"in": "ɛ", "out": "ɛ"},
+    {"in": "h", "out": "h"},
+    {"in": "i", "out": "i"},
+    {"in": "k", "out": "k"},
+    {"in": "l", "out": "l"},
+    {"in": "ɬ", "out": "s"},
+    {"in": "m", "out": "m"},
+    {"in": "n", "out": "n"},
+    {"in": "ɴ", "out": "ŋ"},
```
Comment (roedoejet) [inline g2p/mappings/langs/generated/clm-ipa_to_eng-ipa.json:41]: this handling of engma looks better!

File: g2p/tests/public/data/clm.csv [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+clm,eng-arpabet,sx̣ʷáx̣ʷc̕,S SH W AE SH W T S
+
+# Artificial test data
+clm,clm-ipa,uʔúhuy̕,ɔʔˈɔhɔj̰
+clm,clm-ipa,utúluk,utˈuluk
+clm,clm-ipa,ə,ə
+clm,clm-ipa,ə́,ˈʌ
+clm,clm-ipa,šəč̕əš,ʃɨtʃʼɨʃ
+clm,clm-ipa,čəšəč,tʃɨʃɨtʃ
+clm,clm-ipa,ə́x̣ʷəx̣ʷ,ˈʌχʷʊχʷ
```
Comment (roedoejet) [inline g2p/tests/public/data/clm.csv:42]: nice - glad to see some of these context-sensitive edge cases here, even if they're artificial.

**Review by roedoejet — APPROVED** [review]
Excellent, quick, and thorough work @joanise - looks good to me.

File: g2p/tests/public/data/clm.csv (hunk 3 above)
Comment (joanise) [reply g2p/tests/public/data/clm.csv:42]: Yeah, those rules were tricky to write correctly, these were essential to get them working right in the first place!
